layouts/_default/baseof.html

```diff
@@ -1,23 +1,26 @@
-{{ $lang := site.LanguageCode }}
 <!DOCTYPE html>
-<html{{ with $lang }} lang="{{ . }}"{{ end }}>
-  <head>
-    <title>
-      {{ block "title" . }}
-        {{ site.Title }}
-      {{ end }}
-    </title>
-    {{ partial "css.html" . }}
-    {{ partial "favicon.html" . }}
-  </head>
+<html lang='{{ .Site.LanguageCode }}'>
+    {{- partial "head.html" . -}}
```

Comment:
Can we do two spaces and enforce that in the `.editorconfig` file?

Reply:
I'll switch to VS code for you, Luc ;)